operator-pipeline-images/tests/entrypoints/test_create_container_image.py

```diff
@@ -145,3 +204,36 @@ def test_remove_latest_from_previous_image(mock_put: MagicMock, mock_get: MagicM
             "repositories": [{"tags": [{"name": "some_other"}]}],
         },
     )
+
+    # for no query results
+
+    mock_rsp = MagicMock()
+    mock_get.return_value = mock_rsp
+
+    mock_rsp.json.return_value = {"data": []}
+    remove_latest_from_previous_image(pyxis_url, isv_pid)
+
+
+@patch("operatorcert.entrypoints.create_container_image.setup_argparser")
+@patch("operatorcert.entrypoints.create_container_image.check_if_image_already_exists")
+@patch(
+    "operatorcert.entrypoints.create_container_image.remove_latest_from_previous_image"
+)
+@patch("operatorcert.entrypoints.create_container_image.create_container_image")
+@patch("operatorcert.entrypoints.create_container_image.json.load")
+@patch("operatorcert.entrypoints.create_container_image.open")
+def test_main(
```

Comment:
Just curious... In a bunch of entry points, you ignore the main function with `pragma`. Why do you test this one and ignore others?

Reply:
I forgot to remove these tests for the main function. Initially, I started with writing the unit tests for the main function, then realized that it isn't needed. I will remove them.